.travis.yml

```diff
@@ -1,24 +1,33 @@
-# See https://github.com/silverstripe-labs/silverstripe-travis-support for setup details
-
-sudo: false
-
 language: php
 
+env:
+  global:
+    - COMPOSER_ROOT_VERSION="2.0.x-dev"
+
 matrix:
   include:
-    - php: 5.4
-      env: DB=SQLITE3 CORE_RELEASE=3
-    - php: 5.5
-      env: DB=PGSQL CORE_RELEASE=3
     - php: 5.6
-      env: DB=MYSQL CORE_RELEASE=3
+      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
+    - php: 7.0
+      env: DB=PGSQL PHPUNIT_TEST=1
+    - php: 7.1
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
+    - php: 7.2
+      env: DB=MYSQL PHPUNIT_TEST=1 SUBSITES=1
 
 before_script:
-  - composer self-update || true
-  - git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
-  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
-  - cd ~/builds/ss
-  - composer install
+  - phpenv rehash
+  - phpenv config-rm xdebug.ini
+
+  - composer validate
+  - if [[ $DB == "PGSQL" ]]; then composer require --no-update silverstripe/postgresql 2.0.x-dev; fi
+  - if [[ $SUBSITES ]]; then composer require --no-update silverstripe/subsites 2.0.x-dev; fi
```

Comment:
We don't need subsites in this module, so we can remove this line and the `SUBSITES=1` environment variable on line 16.

Also we need to add `composer require --no-update silverstripe/recipe-cms ^1.0` at about this position too, so that the Page class exists

Reply:
Also a linting error to go, then we should be 🍏  https://travis-ci.org/silverstripe/silverstripe-gridfieldqueuedexport/jobs/330064426#L625